build.rs

```diff
@@ -99,13 +98,13 @@ fn format_test((k, v): (&String, &toml::Value)) -> String {
     format!(r#"printf "{}"
 
 if result=$({}); then
-    echo " $check"
+    echo -e " $check"
```

Comment:
Can we use `printf` instead of `echo -e`?

Reply:
My bash-fu isn't strong enough. I couldn't figure out how to emulate '''echo " $result"' with printf. Using %s print all lines left-aligned.

'''
format ✖ Diff
 in .../main.rs
 at line
 280: let
 sigterm =
 unix::Signal::new(unix::libc::SIGTERM, &handle).flatten_stream();⏎
 let sigint
 = unix::Signal::new(unix::libc::SIGINT,
 &handle).flatten_stream();⏎ ⏎
 - let
 stream =
 sigint.select(sigterm).take(1).for_each(|s| {⏎
 + let
 stream =
 sigint.select(sigterm).take(1).for_each(|s| {⏎
 info!(root_logger, "Got
 signal {:?}",
 s);⏎ ⏎
 Ok(())⏎ 
'''

vs 

'''
format ✖ Diff in .../main.rs at line 280:
     let sigterm = unix::Signal::new(unix::libc::SIGTERM, &handle).flatten_stream();⏎
     let sigint = unix::Signal::new(unix::libc::SIGINT, &handle).flatten_stream();⏎
 ⏎
-        let stream = sigint.select(sigterm).take(1).for_each(|s| {⏎
+    let stream = sigint.select(sigterm).take(1).for_each(|s| {⏎
         info!(root_logger, "Got signal {:?}", s);⏎
 ⏎
         Ok(())⏎

'''